clang/lib/Index/IndexDataStoreUtils.cpp

```diff
@@ -285,6 +285,9 @@ SymbolRoleSet index::getSymbolRoles(uint64_t Roles) {
 indexstore_symbol_kind_t index::getIndexStoreKind(SymbolKind K) {
   switch (K) {
   case SymbolKind::Unknown:
+  case SymbolKind::TemplateTypeParm:
+  case SymbolKind::TemplateTemplateParm:
+  case SymbolKind::NonTypeTemplateParm:
     return INDEXSTORE_SYMBOL_KIND_UNKNOWN;
```

Comment:
Could you also add a `FIXME` comment to indicate that we should figure out the best way to expose this info?